### DISCUSSION TITLE
Resources to learn Kotlin for 2024-2025

### DISCUSSION THREAD

**Discussion by SjxSubham** [community/community#138092] [Programming Help]
### Select Topic Area

Question

## **Give me some good resources to learn Kotlin...** https://user-images.githubusercontent.com/74132293/99877850-13a72580-2c27-11eb-8fb3-d1ebc7e6842d.png
It'll also be helpfull if it will any Indian resource

**Comment by ferid333** [comment]
Hello @SjxSubham ! I can recommend you a couple of resources to start learning Kotlin. 

Kotlin roadmap: https://medium.com/@sandeepkella23/android-kotlin-roadmap-explained-bfb789f5cc3e

Kotlin official docs : https://kotlinlang.org/docs/getting-started.html

Basic concepts: https://www.w3schools.com/KOTLIN/index.php

Comprehensive resource : https://www.geeksforgeeks.org/kotlin-programming-language/

Additionally, I would recommend YouTube videos and online courses. Good luck for that journey

**Reply by SjxSubham** [reply]
if u know any good playlist regarding _Kotlin_**** then, Give

**Reply by ferid333** [reply]
This is good video for learning : https://www.youtube.com/watch?v=EExSSotojVI

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @SjxSubham ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
